tests/components/pglab/test_config_flow.py

```diff
@@ -0,0 +1,113 @@
+"""Test the PG LAB Electronics config flow."""
+
+from homeassistant import config_entries
+from homeassistant.components.pglab.const import DOMAIN
+from homeassistant.core import HomeAssistant
+from homeassistant.helpers.service_info.mqtt import MqttServiceInfo
+
+from tests.common import MockConfigEntry
+from tests.typing import MqttMockHAClient
+
+
+async def test_mqtt_abort_if_existing_entry(
+    hass: HomeAssistant, mqtt_mock: MqttMockHAClient
+) -> None:
+    """Check MQTT flow aborts when an entry already exist."""
+    MockConfigEntry(domain=DOMAIN).add_to_hass(hass)
+
+    result = await hass.config_entries.flow.async_init(
+        DOMAIN, context={"source": config_entries.SOURCE_MQTT}
+    )
+
+    assert result["type"] == "abort"
```

Comment:
```suggestion
    assert result["type"] is FlowResultType.ABORT
```

Reply:
Please use both `is` and the `FlowResultType` enum throughout the config flow tests

Reply:
OK